docs/template.md

```diff
@@ -1,7 +1,25 @@
-# qgrid 
+# qgrid template
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="data" >
+        <q-grid:head> 
+            <q-grid:td> 
+                <i title="{{$column.name}}">{{$column.label}}</i>
+            </q-grid:td>
+        </q-grid:head>
+        <q-grid:body>
+            <q-grid:tr>
```

Comment:
We can provide some pre-defined css classes for different states of row.  Also for row customization we can make soemthing like this:
<q-grid:tr ng-class="{ extraordinaryClass: $row.condition }"></q-grid:tr>